public/util.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const firebaseConfig = {
+  apiKey: 'AIzaSyDzObPF0TEJ4G80DFUnQzRDwy8oNQiTH4k',
```

Comment:
If you will use secrets or not in semaphore - these files will be simply available as /util.js

Reply:
Probably public/util.js and worker/util.js duplicate each other - maybe better to use only one which is not public?